### PR TITLE
Use bazel-build protoc and plugins to regenerate protos for some languages.

### PR DESCRIPTION
**Pull request by jtattermusch**
- with bazel `protoc` and `grpc_*_plugin` are much easier to build
- we want to use `make` as little as possible

This PR also regenerates the out-of-date generated files.

### CODE REVIEW

File: src/objective-c/tests/run_plugin_tests.sh [hunk 1]
```diff
@@ -24,9 +24,8 @@ cd $(dirname $0)
 
 ROOT_DIR=../../..
 BAZEL=$ROOT_DIR/tools/bazel
-BAZEL_EXEC_ROOT=$ROOT_DIR/bazel-out/darwin-fastbuild/bin
-PROTOC=$BAZEL_EXEC_ROOT/external/com_google_protobuf/protoc
-PLUGIN=$BAZEL_EXEC_ROOT/src/compiler/grpc_objective_c_plugin
+PROTOC=bazel-bin/external/com_google_protobuf/protoc
```
Comment (muxi) [inline src/objective-c/tests/run_plugin_tests.sh:27]: You probably still need `$ROOT_DIR` here

Comment (jtattermusch) [reply src/objective-c/tests/run_plugin_tests.sh]: Good catch! Fixed.

**Comment by jtattermusch** [comment]
There's a potential problem with php:
https://source.cloud.google.com/results/invocations/f92bf93c-a053-4ec5-b831-a25b51d5cae6/targets/github%2Fgrpc%2Finterop_test/tests;group=tests;test=cloud_to_cloud:php7:ruby_server:large_unary:tls;row=232

**Comment by jtattermusch** [comment]
Known failures: https://github.com/grpc/grpc/issues/21839

**Comment by jtattermusch** [comment]
I might try to do some followup for php.

**Pull request merged**